addons/website/views/snippets/s_key_benefits.xml

```diff
@@ -4,32 +4,32 @@
 <template id="s_key_benefits" name="Key benefits">
     <section class="s_key_benefits pt48 pb48">
         <div class="container">
-            <div class="row o_grid_mode s_nb_column_fixed" data-row-count="11" style="column-gap: 48px;">
-                <div class="o_grid_item g-col-lg-4 g-height-1 col-lg-4" style="--grid-item-padding-x: 16px; --grid-item-padding-y: 0px; grid-area: 1 / 1 / 2 / 5; z-index: 1;">
+            <div class="row s_nb_column_fixed" data-row-count="11">
```

Comment:
Why preventing the edition of this `.row`? 

I know that the default design we're proposing assumes a fixed number of columns here, but the user may need to deviate and add more... it should be a problem IMO 😉

Reply:
Shouldn't be here indeed, it's a leftover from a time I wanted to preserve a balance layout at all cost 😅